cmd/opdoc/opdoc.go

```diff
@@ -50,6 +51,18 @@ func markdownTableEscape(x string) string {
 	return strings.ReplaceAll(x, "|", "\\|")
 }
 
+func namedStackTypesMarkdown(out io.Writer, stackTypes []namedType) {
+	fmt.Fprintf(out, "#### StackType Definitions\n\n")
+	fmt.Fprintf(out, "| Name | Bound | AVM Type |\n")
+	fmt.Fprintf(out, "| ---- | ---- | -------- |\n")
+
+	for _, st := range stackTypes {
+		bound := fmt.Sprintf("%d - %d", st.Bound[0], st.Bound[1])
+		fmt.Fprintf(out, "| %s | %s | %s |\n", st.Name, bound, st.AVMType)
+	}
```

Comment:
Can we neaten this up a bit?  Avoid printing bounds when there are none? Maybe use " <= X" if the lower bound is 0?  Maybe make it clear we're talking about length in the base of bytes?

I can always clean this up later though, so it's not pressing.